src/index.js

```diff
@@ -60,8 +60,19 @@ const supernova = (env) => {
       useEffect(() => {
         if (!state.mounted && selections !== undefined && layout.qHyperCube) {
           state.mounted = true;
-          const c = pic.chart({ element, settings: {}, data: [] });
-
+          const settings = picassoDef({
+            layout,
+            theme,
+            env,
+            picassoQ,
+            selectionsApi: selections,
+            showLegend,
+            invalidMessage,
+            translator,
+            options,
+          });
+          const data = { type: 'q', data: layout.qHyperCube, key: 'qHyperCube' };
+          const c = pic.chart({ element, settings, data });
```

Comment:
why does it need to render with data here?
It make it harder to add fetching extra async data harder (needed to fix more advanced coloring)

Reply:
Can you point me where this is causing an issue to do async fetches? I would like to understand the problem before I address this further.

Reply:
Done.